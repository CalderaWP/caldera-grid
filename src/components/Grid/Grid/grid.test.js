import React from 'react';
import renderer from 'react-test-renderer';
import {Grid} from "./index";

test('Works with one row', () => {
    const component = renderer.create(
        <Grid
            rows={
                {
                    1: {
                        content: 'HiRoy',
                        uuid: '1'
                    }

                }
            }
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('Works with two rows', () => {
    const component = renderer.create(
        <Grid
            rows={
                {
                    1: {
                        content: 'HiRoy',
                        uuid: '1'
                    },
                    2: {
                        content: 'HiMike',
                        uuid: '2'
                    }
                }
            }

        />

    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
