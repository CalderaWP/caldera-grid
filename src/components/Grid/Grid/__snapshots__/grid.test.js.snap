// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works with one row 1`] = `
<div
  className="calderaGrid"
>
  <div>
    HiRoy
  </div>
</div>
`;

exports[`Works with two rows 1`] = `
<div
  className="calderaGrid"
>
  <div>
    HiRoy
  </div>
  <div>
    HiMike
  </div>
</div>
`;
